plugins/package-managers/sbt/src/funTest/kotlin/SbtFunTest.kt

```diff
@@ -58,14 +59,14 @@ class SbtFunTest : StringSpec({
         // Clean any previously generated POM files / target directories.
         GitCommand.run(definitionFile.parentFile, "clean", "-fd").requireSuccess()
 
-        val ortResult = analyze(
+        val result = analyze(
             definitionFile.parentFile,
             packageManagers = setOf(SbtFactory()),
             packageManagerConfiguration = mapOf(
                 "SBT" to PackageManagerConfiguration(options = mapOf("javaVersion" to "11"))
             )
-        )
+        ).getAnalyzerResult()
 
-        patchActualResult(ortResult.toYaml(), patchStartAndEndTime = true) shouldBe expectedResult
+        patchActualResult(result.toYaml()) shouldBe expectedResult
```

Comment:
Is it correct that `patchActualResult()` is till required here, but not above?

Reply:
Yes. This asset still has one `<REPLACE_*>` pattern in it.